kellydinneen/rancid-tomatillos
# Refactor organization

#### Description
- Reorganize and put images in their own directory
- Refactor App.js functions to be ES6 arrow functions so we don't have to bind them, does so automatically
- Create route for Login and create Link for it with the button already in place from Kelly

#### Type of change

- [x] Housekeeping (e.g. README, file naming)

  - [x] *DRYing*
  - [ ] *SRPifying*
  - [x] *Other:* Moving to ES6 functions

#### Functionality area

- [x] React component

#### Context
- Makes these more organized, and makes for easily readability 
- Makes Login page accessible 

#### Where a reviewer should start
 - App.js line 41

#### How to test this change
- Click Login button and see Login page and the url endpoint change to include '/login'

#### Next Steps/Loose Ends
- Create testing for Login page
- Finish testing for dashboard
- Create 'Make account' option
- Create 'Profile' link/page
